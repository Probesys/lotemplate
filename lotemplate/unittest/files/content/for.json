{
  "tutu": {
    "type": "array",
    "value": [
      {
        "firstName": "toto",
        "lastName": "string",
        "address": {
          "street1": "8 rue de la paix",
          "street2": "",
          "city": "Paris",
          "state": "Ile de France"
        }
      }
    ]
  }
}