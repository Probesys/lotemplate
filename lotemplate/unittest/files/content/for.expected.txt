
Tests of for statements


[for oui]
my paragraph
[endfor]

